src/main/java/Duke.java

```diff
@@ -90,7 +112,7 @@ private static boolean isInt(String str) {
 
     private static void markTask(Task[] listOfTasks, String[] words) {
         int number = Integer.parseInt(words[1]);
```

Comment:
I'm not too sure what this number is referring to... Perhaps giving a name such as `taskNumber` could make it clearer on what `number` refers to.